spark/src/main/scala/org/apache/spark/sql/delta/TableFeature.scala

```diff
@@ -246,6 +280,47 @@ object TableFeature {
   /** Get a [[TableFeature]] object by its name. */
   def featureNameToFeature(featureName: String): Option[TableFeature] =
     allSupportedFeaturesMap.get(featureName.toLowerCase(Locale.ROOT))
+
+  /**
+   * Extracts the removed feature name by comparing current and base protocols.
+   * Returns None if there is no removed feature.
+   * Assumes there can only be one feature removed at a time.
+   */
+  protected def getRemovedFeatureName(
+      currentProtocol: Protocol,
```

Comment:
For all the call sites of this function and other functions that have multiple protocol parameters, please use named arguments. It is very easy to get the order wrong. Especially in this case, where the logic is actually "second argument minus first argument" -- I would intuitively pass the arguments in the wrong order if I weren't paying attention. :)